core/src/test/java/org/elasticsearch/VersionTests.java

```diff
@@ -33,44 +33,43 @@
 import java.util.Map;
 import java.util.Set;
 
-import static org.elasticsearch.Version.V_2_2_0;
-import static org.elasticsearch.Version.V_5_0_0_alpha1;
+import static org.elasticsearch.Version.V_5_3_0_UNRELEASED;
+import static org.elasticsearch.Version.V_6_0_0_alpha1_UNRELEASED;
 import static org.elasticsearch.test.VersionUtils.randomVersion;
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.greaterThan;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.lessThan;
-import static org.hamcrest.Matchers.lessThanOrEqualTo;
 import static org.hamcrest.Matchers.sameInstance;
 
 public class VersionTests extends ESTestCase {
 
     public void testVersionComparison() throws Exception {
-        assertThat(V_2_2_0.before(V_5_0_0_alpha1), is(true));
-        assertThat(V_2_2_0.before(V_2_2_0), is(false));
-        assertThat(V_5_0_0_alpha1.before(V_2_2_0), is(false));
+        assertThat(V_5_3_0_UNRELEASED.before(V_6_0_0_alpha1_UNRELEASED), is(true));
```

Comment:
I think we should (in a separate issue) change these tests to not rely on constants. They need constant maintenance because of this, when in reality the logic here can be checked against versions created by parsing.